lib/services/registered_addons.pm

```diff
@@ -64,20 +64,26 @@ sub check_upgraded_addons {
 }
 
 sub check_suseconnect {
-    my $output = script_output("SUSEConnect -s");
-    my $json   = Mojo::JSON::decode_json($output);
-    foreach (@$json) {
-        my $iden   = $_->{identifier};
-        my $status = $_->{status};
-        die "$iden register status is: $status" if ($status ne 'Registered');
+    my $output = script_output("SUSEConnect -s", 120);
```

Comment:
why not just `assert_script_run 'SUSEConnect --status-text | (! grep -q "Not Registered")';` instead of the whole block?

Reply:
I think we can use this way to get more exact result. If any addons status is not registered, then it can show up the addons' name. It is easy for reviewer get the failed information and report the bug.

Reply:
I see